### PR TITLE
CDAP-15864. Two tests with file plugins were added. Moved from Dataproc cluster env to sandbox default.

### PR DESCRIPTION
**Pull request by dvitiiuk**
JIRA: https://issues.cask.co/browse/CDAP-15864
DOC: https://wiki.cask.co/display/CE/Google+Drive+plugins

### CODE REVIEW

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java [hunk 1]
```diff
@@ -89,6 +108,7 @@
   private static final String TEST_TEXT_FILE_CONTENT = "text file content";
   private static final String TEST_DOC_FILE_CONTENT = "Google Document file content";
   private static final String TEST_SHEET_FILE_CONTENT = "a,b,c\r\n,d,e";
+  public static final String TMP_FOLDER = "/tmp/googleDriveTestFolder";
```
Comment (aonischuk) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java]: cdap can run on windows. Please use
`Files.createTempDirectory` instead. Also don't forget to close it.

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java [hunk 2]
```diff
@@ -390,7 +420,162 @@ public void testPartitionSize() throws Exception {
...
+      String fileContent = String.join(",", fileLines);
+      JsonElement rootElement = jsonParser.parse(fileContent);
+      Assert.assertTrue(rootElement.isJsonObject());
+
+      JsonObject rootObject = rootElement.getAsJsonObject();
+      Set<Map.Entry<String, JsonElement>> entries = rootObject.entrySet();
+
+      int resultOffset = 0;
+      byte[] resultBody = new byte[]{};
+      for (Map.Entry<String, JsonElement> entry : entries) {
```
Comment (aonischuk) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java]: why do we need loop here? Can't we just do: rootObject.entrySet().get("name") rootObject.entrySet().get("mimeType") etc.
and than check number of keys.

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java [hunk 3]
```diff
@@ -16,6 +16,7 @@
 
 package io.cdap.cdap.app.etl.google.drive;
 
+import static org.junit.Assert.assertArrayEquals;
```
Comment (aonischuk) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java]: Please remove ALL static imports. Our code style does not allow that.

**Review by aonischuk — CHANGES_REQUESTED** [review]

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java (hunk 2 above)
Comment (dvitiiuk) [reply integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java]: done

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java (hunk 1 above)
Comment (dvitiiuk) [reply integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java]: done

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java (hunk 3 above)
Comment (dvitiiuk) [reply integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java]: done

**Pull request merged**